doc/abandon.md

```diff
@@ -0,0 +1,105 @@
+# Abandon
+
+## Command line arguments
+
+ - '-g' := Start graphical GUI
+ - '-c' <conf-file> := path to journal's configuration file
+ - '-i' <input-file> := input file, could be specified inside conf-file
+ 
+## Configuration file
+
+
+### Input -directive
+
+Input directive defines primary inputs for this particular journal. 
+There could be other "include" directives inside these primary inputs.
+
+Input directive supports following formats:
+
+	input += input
+
+where input could be:
+ 
+ - file-path, this path could be absolute, or relative to the conf-file
+ - "glob:" + pattern, this is glob based pattern matching, see below for further info
+ - "regex:" + pattern, this is regex based pattern matching, see below for further info
+      
+#### Wildcard support
+
+With both wildcard inputs (`glob:` and `regex`) candidates files paths are matched 
+against filenames which are full, absolute paths to those files.
+
+Input files are unique set, so if a file gets included multiple times by wildcard pattern,
+it will be used only once. 
+
+##### Glob syntax
+
+Glob syntax supported by Adandon is based on Java's NIO Glob based filename matching.
```

Comment:
Oh, so you are familiar with `NIO`. My previous remark probably doesn't apply then.

I like this documentation effort very much! :clap: